app/controllers/admin/commits_controller.rb

```diff
@@ -0,0 +1,11 @@
+class Admin::CommitsController < ApplicationController
+  
+  def index
+    from = params[:from].presence.try(:to_date).try(:beginning_of_day) || Date.yesterday.beginning_of_day
+    to = params[:to].presence.try(:to_date).try(:end_of_day) || Date.yesterday.end_of_day
+    @commits = Commit.all
```

Comment:
leave a line between  fetching commits and to/from

Reply:
okay sir